emails/2022-11-11.md

```diff
@@ -0,0 +1,40 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+We've found that the best to describe Dolt is: It's like Git and MySQL had a baby. I often wonder what a mascot for such a thing would look like. MySQL is a Dolphin. GitHub is an Octocat. Is Dolt an Octodolphin?
+
+[![Octodolphin](../images/octodolphin.png)](https://www.doltdb.com)
+
+Who needs designers? THe new Dolt logo is perfect.
+
+### Dolt now supports every MySQL type
+
+Some of you may know, we have a [whole section of our documentation on MySQL compatibility](https://docs.dolthub.com/sql-reference/sql-support). The documentation is divided into sections regarding SQL features like indexes, functions, and schema. If Dolt matches MySQL functionality, we put a green checkmark in the table. If we mostly match, we either put a caveat in the third column, or mark it yellow instead of check. Finally, if we don't support it, we put the dreaded red x. Here is an example.
+
+[![Table example](../images/sql-support-docs.png)](https://docs.dolthub.com/sql-reference/sql-support/data-description)
+
+A couple weeks ago, we turned an entire table green checkmark, [the types table](https://docs.dolthub.com/sql-reference/sql-support/data-description#data-types)! I know to you all, you are like "cool story". But to us, we started Dolt with no SQL support. We hand-crafted our own SQL engine before we found [go-mysql-server](https://github.com/dolthub/go-mysql-server), ie. GMS. When the company supporting GMS went out of business we took over support for their open source project. We've spent three years improving it.  We want to turn every table all green checkmark and I'm really happy we got our first. This has been a journey and this feels like a nice oasis to stop at and reflect.
```

Comment:
Maybe say this matters bc we want Dolt to be a dropin replacement. no coding, etc. you just get to be creative with a version control database without have to learn anything new assuming you know git and mysql